src/main/java/org/folio/consortia/service/impl/SharingInstanceServiceImpl.java

```diff
@@ -80,7 +79,11 @@ public SharingInstance start(UUID consortiumId, SharingInstance sharingInstance)
       }
 
       try (var context = new FolioExecutionContextSetter(prepareContextForTenant(targetTenantId, folioModuleMetadata, folioExecutionContext))) {
-        String source = FOLIO_SOURCE_VALUE.equalsIgnoreCase(inventoryInstance.get("source").asText()) ? CONSORTIUM_FOLIO : CONSORTIUM_MARK;
+        String source = switch (inventoryInstance.get("source").asText()) {
+          case "folio" -> CONSORTIUM_FOLIO;
```

Comment:
theretically souce can come with upper registry, so safer to compare with equalsIgnoreCase

Reply:
Fixed by using .toLowerCase method of string